### PR TITLE
Simplify migration stuff

### PR DESCRIPTION
**Pull request by bliuchak**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,65 +1,5 @@
...
-	docker-compose create flat-search
-	docker-compose stop
-
-docker-start:
-	docker-compose start
-
-docker-stop:
-	docker-compose stop
+migrate-up:
+	docker run -v "${CURDIR}/migrations:/migrations" --network host migrate/migrate -path=/migrations/ -database "mysql://root:test100500@tcp(localhost:3306)/flat-search" up
```
Comment (grubastik) [inline Makefile:2]: Why did you remove all the other things from make file?

Comment (grubastik) [inline Makefile:2]: What if someone will run DB on a separate host or in another docker container or maybe on different port?

File: aggregator/builder.go [hunk 2]
```diff
@@ -40,6 +41,9 @@ func ProcessAdverts(c *config.Sreality) error {
 		}
 		l = append(l, ConvertSrealityToModel(a, *d))
 	}
+
+	fmt.Println("processed adverts:", len(l))
```
Comment (grubastik) [inline aggregator/builder.go:45]: What the point to add here this code? Is it related to migrations?

File: docker-compose-migration.yml [hunk 3]
```diff
@@ -1,17 +0,0 @@
-version: '2'
```
Comment (grubastik) [inline docker-compose-migration.yml:1]: What the point to remove this docker file?

File: main.go [hunk 4]
```diff
@@ -2,21 +2,33 @@ package main
...
+
+	email.NewConnection(cfg)
 
-	err = aggregator.ProcessAdverts(config.GetSreality())
-	error.DebugError(err)
+	err = aggregator.ProcessAdverts(cfg.GetSreality())
+	if err != nil {
+		log.Fatal(err, "process adverts error")
+	}
+	log.Println("end")
```
Comment (grubastik) [inline main.go:33]: Is all these changes related to migrations?

File: Makefile (hunk 1 above)
Comment (bliuchak) [reply Makefile:2]: because they are useless

Comment (bliuchak) [reply Makefile:2]: this solution won't fit your requirement. in the future the feature you're talking about might be added if necessary

File: main.go (hunk 4 above)
Comment (bliuchak) [reply main.go:33]: i just wanted to know when and where app stops

File: docker-compose-migration.yml (hunk 3 above)
Comment (bliuchak) [reply docker-compose-migration.yml:1]: because my solution don't need them

File: aggregator/builder.go (hunk 2 above)
Comment (bliuchak) [reply aggregator/builder.go:45]: it's not related. just adding some info to the app output
